labeler.yml

```diff
@@ -0,0 +1,27 @@
+name: Add Hacktoberfest label on PR approval
+
+on:
+  pull_request_review:
+    types: [submitted]
+  pull_request:
+    types: [closed]
+
+jobs:
+  add_label_on_approval:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Add Hacktoberfest label
+        if: github.event_name == 'pull_request_review' && github.event.review.state == 'approved' && github.event.pull_request.labels == null && github.event.pull_request.mergeable_state == 'clean'
+        uses: actions/github-script@v4
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.issues.addLabels({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.payload.pull_request.number,
+                labels: ['Hacktoberfest']
```

Comment:
the label should be 'hacktoberfest-accepted'

Reply:
okay